mkdocs.yml

```diff
@@ -102,6 +103,6 @@ plugins:
 
 # Preview controls
 watch:
-  - src
+  - "src"
```

Comment:
Because it is generated via a template and since we don't know the content 100% sure, we quote it. We don't need to have the inconsistency though, but it should be reported in repo-config, as the template should be fixed and then this repo updated via the template.